src/azul/logging.py

```diff
@@ -163,20 +175,59 @@ def silenced_es_logger():
             assert es_log.level == original_log_level
 
 
-def http_body_log_message(body_type: str,
-                          body: bytes | bytearray | str | IO[bytes] | IO[str] | None,
+type BodyType = Literal['request', 'response']
+
+first_kibibit = 1024
+
+
+def http_body_log_message(body_type: BodyType,
+                          body: bytes | bytearray | str | IO[bytes] | IO[str] | JSON | None,
                           *,
-                          verbatim: bool = False,
+                          debug: int = 1,
                           ) -> str:
+    """
+    Returns a string containing the body_type (request or response) in addition
```

Comment:
First sentence is incomprehensible. 

The second sentence is brittle. You don't want to have to change the docstring when you change the variable. Do we still need the variable?

I don't think obfuscating "1024 bytes" as "KiB" and "Kibibyte" is helpful in this context.